win32ss/user/user32_vista/user32_vista.spec

```diff
@@ -0,0 +1 @@
+@ stdcall GetDpiForWindow(ptr)
```

Comment:
Add a `-version=0xA00+`, at least as a comment, as Vista does not have this function.